.buildkite/pipeline.yml

```
@@ -74,8 +74,9 @@ steps:
       buildkite-agent artifact download "*.nupkg" .
       buildkite-agent artifact download "*\\RELEASES" .
 
-      echo "--- :node: Generating Release Manifest"
       .buildkite/commands/install-node-dependencies.sh
```

I wonder if we could include this command in its own section:
```suggestion
      echo "--- :node: Installing Node dependencies"
      .buildkite/commands/install-node-dependencies.sh
```

---

We could, but the `.buildkite/commands/install-node-dependencies.sh` [already prints its own sections too](https://github.com/Automattic/studio/blob/trunk/.buildkite/commands/install-node-dependencies.sh#L18), so not sure that's necessary to add one more in practice.

Alternatively, we could use `--- :node: Installing Node dependencies` here and use `~~~` instead of `---` for the logs inside the `install-node-dependencies.sh` script, hoping that those `~~~` would be make them be interpreted as "sub-sections"? (see https://buildkite.com/docs/pipelines/managing-log-output, though I'm not 100% sure what they mean by "and de-emphasize" in that context)

---

> We could, but the `.buildkite/commands/install-node-dependencies.sh` [already prints its own sections too](https://github.com/Automattic/studio/blob/trunk/.buildkite/commands/install-node-dependencies.sh#L18), so not sure that's necessary to add one more in practice.

Ah, true. I overlooked that the script was already including it. In that case, forgot my suggestion.